### PR TITLE
[GH-88] rename to Verdaccio instead of Sinopia

### PR DESCRIPTION
**Pull request by kgrubb**
Based on issue #88

__CHANGES:__

* Aside from a few locations (such as links to issues & the history), 'sinopia' has been renamed to  'verdaccio'.

* The [.travis.yml](https://github.com/kgrubb/verdaccio/blob/master/.travis.yml) has been updated to run on officially supported node versions (I can switch this back if needed).

### CODE REVIEW

**Comment by Nighthawk22** [comment]
This will be a breaking change. No old db will be working without renaming and also no sinopia plugin

**Comment by kgrubb** [comment]
Right, I'll add some sinopia plugin compatibility into this then. What about db data, though? Is that simply changing the `.verdaccio-db.json` back to `.sinopia-db.json`?

**Comment by kgrubb** [comment]
Alright, sinopia plugin compatibility seems fine again. I think this should also fix issue #64 as well now.

**Comment by kgrubb** [comment]
Is it required to get everything passing on node version 0.12, even though it is no longer a supported release? Tests pass on 4, 6, & 7, but not for 0.12. I'd prefer to drop 0.12 from the travis testing, but I'm unsure of how many people are still using 0.12 to deploy/use verdaccio.

**Comment by juanpicado** [comment]
Actually 0.12 were dropped recently, minimum is >=4.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
+  - '0.12'
```
Comment (juanpicado) [inline .travis.yml]: Please, remove this part in order to merge.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
   - '4'
   - '6'
-  - '7'
+  - 'iojs'
```
Comment (juanpicado) [inline .travis.yml]: Please, remove this part in order to merge.

**Comment by juanpicado** [comment]
Looks good to me, please fix the issues and I'll merge

**Comment by juanpicado** [comment]
Thanks @kgrubb 👍

**Pull request merged**